finalProject/highLevelCC.js

```diff
@@ -0,0 +1,64 @@
+//This code in not completed and is still a work in progress. 
+class Letters {
+  constructor(posVal,negVal) {
+    this.posVal = posVal;
+    this.negVal = negVal;
+  }
+  var a = new Letter(0,-26);
```

Comment:
Keep exploring strategies here... you might see how array indexing can help you keep track of how the alphabet is ordered and make shifts.